lang/tests/src/test/scala/com/wavesplatform/lang/v1/estimator/FunctionComplexityTest.scala

```diff
@@ -0,0 +1,63 @@
+package com.wavesplatform.lang.v1.estimator
+import com.wavesplatform.DocSource
+import com.wavesplatform.common.utils.EitherExt2
+import com.wavesplatform.lang.Common.NoShrink
+import com.wavesplatform.lang.directives.{DirectiveDictionary, DirectiveSet}
+import com.wavesplatform.lang.directives.values._
+import com.wavesplatform.lang.utils._
+import com.wavesplatform.lang.v1.compiler.Terms
+import com.wavesplatform.lang.v1.compiler.Terms.FUNCTION_CALL
+import com.wavesplatform.lang.v1.estimator.v3.ScriptEstimatorV3
+import com.wavesplatform.lang.v1.evaluator.ctx.BaseFunction
+import com.wavesplatform.lang.v1.traits.Environment
+import org.scalatest.exceptions.TestFailedException
+import org.scalatest.{Matchers, PropSpec}
+import org.scalatestplus.scalacheck.{ScalaCheckPropertyChecks => PropertyChecks}
+
+class FunctionComplexityTest extends PropSpec with PropertyChecks with Matchers with NoShrink {
+  val directives: Iterable[DirectiveSet] =
+  DirectiveDictionary[StdLibVersion].all
+      .flatMap(
+        version =>
+          Seq(
+            DirectiveSet(version, Account, Expression).explicitGet(),
+            DirectiveSet(version, Asset, Expression).explicitGet()
+          ) ++ (if (version >= V3) Seq(DirectiveSet(version, Account, Expression).explicitGet())
```

Comment:
if (version >= V3) than merge Seq of DirectiveSet with DApp content type like this `Seq(DirectiveSet(version, Account, DApp)`

Reply:
Done